Cargo.toml

```diff
@@ -13,7 +13,8 @@ edition = "2018"
 build = "build.rs"
 
 [features]
-default = ["std"]
+default = ["std", "utf8_in_path"]
+utf8_in_path = []
```

Comment:
Let's not bother with a Cargo feature, it'll just be allowed, since I can tell from operators and also looking in other libraries, this is a thing that just happens.